### ISSUE TITLE
Double Commander not responding when regaining lost focus.

### ISSUE THREAD

**Issue by sztejkat** [doublecmd/doublecmd#1717]
### Version

Double Commander
Version: 1.1.3 beta
Revision: 100
Commit: 42dfbf544
Build date: 2023/10/04
Lazarus: 2.2.6.0
FPC: 3.2.2
Platform: x86_64-Linux-qt5
OS version: Ubuntu 23.10 (Kubuntu, KDE)

Widgetset library: Qt 5.15.10, libQt5Pas 5.6.1

### Description

Frequently enough to be a annoying, but not always, the Double Commander freezes for few seconds (not responding to mouse nor keyboard input) when it looses and then gains back the focus (ie. alt-tab from it and backwards). The freeze takes about 1...2 seconds and frequently relates to some activity on discs caused by other applications running in the background (ie. LVM syncing mirrors). The "auto-refresh" is disabled. Sadly I couldn't find a pattern to reproduce this effect with any kind of certainty, so I know I can't call this a usable bug report. 

Any hint how to mitigate it or trace down the cause would be appreciated.

Thanks for good work, I like this program quite a lot.

**Comment by lprot** [comment]
@sztejkat do you reproduce it on the latest version as well? https://doublecmd.sourceforge.io/site/eng/snapshots.php

**Comment by sztejkat** [comment]
Yes ([appimage](https://download.opensuse.org/repositories/home:/Alexx2000:/doublecmd-svn/AppImage/doublecmd-qt-latest-x86_64.AppImage)) , it behaves the same way. The "freeze" seems to be a tiny bit shorter, but it is still there. It is hard to believe that it has anything to do with the amount of work the program performs, as the machine I have is a hell fast bastard. 
Unfortunately I can't figure out a detailed pattern of what I have to do to expose the problem. It usually happens when I keep doublecommander open in the background, with one LVM volume on the left, and other LVM volume content on the right. I edit a lot of files in text editor and alt-tab into double commander to do some file operations. This however doesn't happen always.

Double Commander
Version: 1.2.0 alpha
Revision: 11385
Commit: 55da326
Build date: 2024/06/27
Lazarus: 2.2.6.0
FPC: 3.2.2
Platform: x86_64-Linux-qt5
OS version: Ubuntu 23.10

Widgetset library: Qt 5.15.10, libQt5Pas 5.6.1

Is there a way to run double commander with detailed logging enabled which might help You to figure out the problem?

**Comment by sztejkat** [comment]
A brief testing have shown that gtk versions seems to not have this problem.

Double Commander
Version: 1.2.0 alpha
Revision: 11385
Commit: 55da326
Build date: 2024/06/27
Lazarus: 2.2.6.0
FPC: 3.2.2
Platform: x86_64-Linux-gtk2
OS version: Ubuntu 23.10

Widgetset library: GTK 2.24.33

**Comment by chrapuch** [comment]
GTK version is not for me on KDE, so I spent some time investigating options and found a possible workaround (image).
Options -> File operations -> Show operations progress initially in -> **minimized separate window**
![image](https://github.com/user-attachments/assets/29eac08f-037d-4ede-88da-a938da7c8710)

**Comment by albmat13** [comment]
Same problem on Windows 11:

Double Commander
Version: 1.2.0 alpha
Revision: 11423
Commit: c159f39a8
Build date: 2024/08/27
Lazarus: 3.4.0.0
FPC: 3.2.2
Platform: x86_64-Win64-win32/win64
OS version: Windows 11 22H2 x86_64

**Comment by rehau** [comment]
Had the same on Windows.
Try to delete the file tabs.xml in AppData/Roaming/doublecmd. It seems to grow forever and makes it slow then.
